modules/SofaGeneralLoader/SphereLoader.cpp

```diff
@@ -49,6 +49,12 @@ SphereLoader::SphereLoader()
 {
     addAlias(&d_positions,"sphere_centers");
     addAlias(&d_scale, "scale3d");
+
+    addUpdateCallback("updateTransformPosition", { &d_translation, &d_rotation, &d_scale}, [this](const core::DataTracker&)
+    {
+        reinit();
+        return sofa::core::objectmodel::ComponentState::Valid;
+    }, { });
```

Comment:
In general there shoud have output data field in the last pair of braces {}.

Reply:
:+1: